models/articles.models.js

```diff
@@ -32,4 +23,14 @@ exports.fetchArticleById = (article_id) => {
             }
             return article;
         })
+}
+
+exports.checkArticleExists = (article_id) => {
+    return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .then(({rows}) => {
+        if(!rows.length){
+            return Promise.reject({status: 404, msg: "Article Not Found"})
+        }
```

Comment:
Look pretty much the same as fetchArticleById

Reply:
refactored ✅  thanks 😄